mmv1/products/iap/Settings.yaml

```diff
@@ -0,0 +1,289 @@
+              The possible values are:
+
+              * `POLICY_TYPE_UNSPECIFIED`: Default value. This value is unused.
+              * `MINIMUM`: This policy acts as a minimum to other policies, lower in the hierarchy.
+              		   Effective policy may only be the same or stricter.
+              * `DEFAULT`: This policy acts as a default if no other reauth policy is set.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'allowedDomainsSettings'
+        description: |
+          Settings to configure and enable allowed domains.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'domains'
+            description: |
+              List of trusted domains.
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::Boolean
+            name: 'enable'
+            description: |
+              Configuration for customers to opt in for the feature.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'workforceIdentitySettings'
+        description: |
+          Settings to configure the workforce identity federation, including workforce pools
+          and OAuth 2.0 settings.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'workforcePools'
+            max_size: 1
+            description: |
+              The workforce pool resources. Only one workforce pool is accepted.
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::NestedObject
+            name: 'oauth2'
+            description: |
+              OAuth 2.0 settings for IAP to perform OIDC flow with workforce identity
+              federation services.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'clientId'
+                description: |
+                  The OAuth 2.0 client ID registered in the workforce identity
+                  federation OAuth 2.0 Server.
+              - !ruby/object:Api::Type::String
+                name: 'clientSecret'
+                sensitive: true
+                ignore_read: true
+                description: |
+                  Input only. The OAuth 2.0 client secret created while registering
+                  the client ID.
+              - !ruby/object:Api::Type::String
+                name: 'clientSecretSha256'
+                output: true
+                sensitive: true
+                description: |
+                  Output only. SHA256 hash value for the client secret. This field
+                  is returned by IAP when the settings are retrieved.
+      - !ruby/object:Api::Type::Array
+        name: 'identitySources'
+        description: |
+          Identity sources that IAP can use to authenticate the end user. Only one identity source
+          can be configured. The possible values are:
+
+          * `IDENTITY_SOURCE_UNSPECIFIED`: IdentitySource unspecified. When selected, IAP relies on
```

Comment:
`xxxx_UNSPECIFIED` is meaningless and can be removed?

Reply:
Removed `UNSPECIFIED`